szperacz/logger.py

```diff
@@ -0,0 +1,19 @@
+
+
+class Logger:
+
+    def debug(self, msg_d):
+        return print(f'{DEBUG} {msg_d}')
+
+    def info(self, msg_i):
+        return print(f'{INFO} {msg_i}')
+
+    def error(self, msg_e):
+        return print(f'{ERROR} {msg_e}')
+
+
+DEBUG = '[DEBUG]'
+INFO = '[INFO]'
+ERROR = '[ERROR]'
+
+logger = Logger()
```

Comment:
Good!

Check this also for some additional insights: https://docs.python.org/3/tutorial/modules.html#importing-from-a-package